tools/sense_studio/templates/testing.html

```diff
@@ -0,0 +1,113 @@
+
+{% block title %} Testing {% endblock %}
+
+{% block navigation %}
+    {% include 'navigation.html' %}
+{% endblock navigation %}
+
+{% block settings %}
+    {% include 'settings.html' %}
+{% endblock settings %}
+
+{% block main %}
+<div class="uk-container uk-container-xsmall">
+    <h1 class="uk-heading-medium uk-margin-large-top uk-margin-remove-bottom">Testing</h1>
+    <div class="uk-text-meta uk-text-large uk-margin-medium-bottom">
+        <span uk-icon="icon: folder"></span> {{ path }}
+    </div>
+    <form class="uk-form-stacked" method="post">
+        <input type="hidden" id="project" name="project" value="{{ project }}">
+        <input type="hidden" id="path" name="path" value="{{ path }}">
+
+        <div class="uk-child-width-1-1@s uk-grid-small" uk-grid>
+            <div>
+                <label class="uk-form-label" for="classifier">Custom Classifier</label>
+                <label class="uk-form-label uk-text-danger" id="classifierLabel"></label>
+                <div class="uk-form-controls">
+                    <div class="uk-inline uk-width-1-1">
+                        <span class="uk-form-icon" uk-icon="icon: folder"></span>
+                        <input class="uk-input path-search" type="text" id="classifier" placeholder="/path/to/custom-classifier"
+                               autocomplete="off" oninput="getPathInputs('classifierLabel', 'classifier');" required>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="uk-child-width-1-1@s uk-grid-small" uk-grid>
+            <div>
+                <label class="uk-form-label" for="inputVideoPath">Input video path</label>
+                <label class="uk-form-label uk-text-danger" id="inputVideoPathLabel"></label>
+                <div class="uk-form-controls">
+                    <div class="uk-inline uk-width-1-1">
+                        <span class="uk-form-icon" uk-icon="icon: video-camera"></span>
+                        <input class="uk-input file-path-search" type="text" id="inputVideoPath" placeholder="/path/to/input-video"
+                               autocomplete="off" oninput="getPathInputs('inputVideoPathLabel', 'inputVideoPath');">
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="uk-child-width-1-3@s uk-grid-small" uk-grid>
+            <div>
+                <label class="uk-form-label" for="outputFolder">
+                    Output folder ({{ project }}/...)
+                </label>
+                <div class="uk-form-controls">
+                    <div class="uk-inline uk-width-1-1">
+                        <span class="uk-form-icon" uk-icon="icon: folder"></span>
+                        <input class="uk-input" type="text" id="outputFolder" placeholder="Output Folder"
+                               autocomplete="off">
+                    </div>
+                </div>
+            </div>
```

Comment:
I added this `output_folder` input taker to make saving video convenient for users. Otherwise, in traditional approach it will ask user to give whole path to the video save directory which I think was extra effort. 
So I divided original `path_out` argument from our old script into `output_folder` and `output_video_name` to make it easier.